src/flake8_to_ruff/isort.rs

```diff
@@ -0,0 +1,30 @@
+//! Extract isort configuration settings from a pyproject.toml.
+
+use std::path::Path;
+
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+
+/// The [isort configuration](https://pycqa.github.io/isort/docs/configuration/config_files.html).
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize, Default)]
+pub struct Isort {
+    #[serde(alias = "src-paths", alias = "src_paths")]
+    pub src_paths: Option<Vec<String>>,
+}
+
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize)]
+struct Tools {
+    isort: Option<Isort>,
+}
+
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize)]
+struct Pyproject {
+    tool: Option<Tools>,
+}
```

Comment:
Should we consolidate these with the same structs defined in `black.rs` too?

Reply:
Yeah we might as well. We can have a single `Pyproject` and `Tools`, with `Isort` and `Black` structs defined in their respective files.

Reply:
Consolidated here: https://github.com/charliermarsh/ruff/pull/2082/commits/0780fa0f4c66a6fc5fbb0bc0f1fd2f474db2273f